### DISCUSSION TITLE
[RFC] global.css 내에 reset css 항목 제거

### DISCUSSION THREAD

**Discussion by eunsonny-wanted** [wanteddev/montage-web#23] [RFC]
### 목표

1. global.css 내에 스타일을 reset 하는 항목들을 제거하여 각 프로젝트 별로 자유롭게 reset.css를 세팅할 수 있도록 변경

### 배경

`wds/global.css` 파일 내부에는 각 html 태그들의 스타일을 일부 초기화하는 reset css의 내용들이 포함되어 있는데요. 이 reset css의 내용들이 기존 원스에서 사용하던 reset css의 내용과는 다소 다른 부분이 있어, `wds/global.css`을 import하면 디자인이 깨지는 현상이 있습니다. 🥲
일단 `wds/global.css`에서 필요한 속성만 발라내 별도의 css 파일을 만들어 대체 할 순 있었지만, 신규 프로젝트가 아니라 기존에 진행되던 프로젝트들이라면 같은 이슈를 마주칠 확률이 높을 것 같습니다.

### 제안

reset css의 내용을 global.css와 별도의 파일로 분리 및 export해서 옵셔널하게 사용할 수 있도록 하는건 어떨까요? 🤔

**Comment by Sh031224** [comment]
https://github.com/wanteddev/wds/blob/main/packages/wds/package.json#L16

`theme.css` 를 이용하시면 될 것 같아요!

다만, wds의 reset에 맞추어서 컴포넌트들이 제작되었기 때문에 일부 동작이 달라질 수 있어요
